lib/pact_broker/api/resources/labels.rb

```diff
@@ -28,6 +32,15 @@ def to_json
           )
         end
 
+        private
+
+        def schema
+          if request.get?
```

Comment:
`request.get?` is already being checked in `malformed_request?` but otherwise looks good to me

Reply:
That was the format being followed in other places